php_fpm/README.md

```diff
@@ -40,6 +40,52 @@ Configuration Options:
 
 [Restart the Agent][3] to start sending PHP-FPM metrics to Datadog.
 
+#### Multiple pools
+
+It is also possible to monitor multiple PHP-FPM pools using the same proxy server, a common scenario when running on Kubernetes.
+
+To do so, modify each instance's `php-fpm.conf`. Here is an example:
+
+Instance 1
+
+```
+...
+pm.status_path = /status1
```

Comment:
are `/status1` and `/ping1` placeholders? if so they should have the following format `/<STATUS_1_PATH>` and `/PING_1_PATH>`